### DISCUSSION TITLE
How to forbid the program to be opened multiple times, only open once

### DISCUSSION THREAD

**Discussion by jsamtsang** [tauri-apps/tauri#6765] [Q&A] [answered]
**I want once the program is opened and run, it's not allowed to run again. How to do?**

**Comment by FabianLars** [comment, marked as answer]
We have a plugin for this: https://github.com/tauri-apps/plugins-workspace/tree/dev/plugins/single-instance

**Reply by 2gn** [reply]
It's now in https://github.com/tauri-apps/plugins-workspace/tree/v1/plugins/single-instance
